apps/events.md

```diff
@@ -153,23 +234,62 @@ checkoutState: null
 checkoutUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/1/Baskets/46072&ChangeAction=PickupBasket&PickupToken=MjNiZThlOGM0MjAzZWQ0N2ZjYmZmZDFiZjI3OTQxMzkwOWY4ZjZlOGE4NTM0ZWIxMjg3NmY2NzBiY2IxYzQ1OV8xNDgyMjIzMDcz"
 coupon: null
 couponCampaign: null
+customerEmail: null
 grandAmount: "76,01 €"
 grandAmountNote: "components.productComponent.priceExclusiveVat"
+grandTotal: {amount: 76.01, currency: "EUR", formatted: "€76,01"}
```

Comment:
Please note that this example does not include the tax type (as stated in https://epages.atlassian.net/browse/UC-3025) as I didn't get that part when trying out the event myself.